### DISCUSSION TITLE
Weekly time Budget

### DISCUSSION THREAD

**Discussion by richardklose** [kimai/kimai#2925] [Questions & Answers]
Is there any possibility to set weekly time budgets? I can only select monthly time budget type and could find any setting for adding other time budget types.

**Comment by kevinpapst** [comment]
Nope, unfortunately not yet. 
Adding weekly budgets is quite a challenging task, so I am always looking for financial support by companies that need such a feature.

**Comment by fulldecent** [comment]
Please see a feature specification for weekly time budgets here: https://github.com/kimai/kimai/issues/4730

---

I think a prerequisite to weekly budgets is a comprehensive and clear management of time zone settings across the system. (Also discussed in https://github.com/kimai/kimai/issues/4730

---

My company will consider to contribute money and/or code to support this feature development if we adopt Kimai.
